src/Urbit.ts

```diff
@@ -97,7 +103,7 @@ export class Urbit {
 
   /** This is basic interpolation to get the channel URL of an instantiated Urbit connection. */
   private get channelUrl(): string {
-    return `${this.url}/~/channel/${this.uid}`;
+    return `${this.url}/~/channel-jam/${this.uid}`;
   }
 
   private get fetchOptions(): any {
```

Comment:
on line 111 we still have 'Content-Type': 'application/json', even though we are sending jammed nouns. Is this intentional?

Reply:
Good catch, that should become `application/x-urb-jam`.

Other parts of this PR are outdated as well (wrt the released version from urbit/urbit#6472), like the channel URLs now being the same as normal, and only that `Content-Type` header indicating the difference.